account_bank_statement_import_txt_xlsx/models/account_bank_statement_import_sheet_mapping.py

```diff
@@ -147,6 +174,30 @@ class AccountBankStatementImportSheetMapping(models.Model):
         string='Bank Account column',
         help='Partner\'s bank account',
     )
+    with_metadata = fields.Boolean(
+        string='File wWth Metadata ',
+        help='Check if file containt meta data in first lines',
+    )
+    header_lines_number = fields.Integer(
+        string='Header lines number',
+        help='Set the Header lines number.'
+             'Used in some csv file that integrate meta data in'
+             'first lines. This number contain the number of'
+             'the all meta data lines including columns names',
+        default="1",
```

Comment:
`"` vs `'`

Reply:
I attempted to apply pre-commit check to apply a good practice but it causes a lot of small changes. Which dont simplify code review. In v 13 it will be better for this aspect.

Reply:
Absolutely! Yet if file uses ' initially, I'd keep that as ' for consistency